test/es2015/module-support-assessment-test.mjs

```diff
@@ -2,13 +2,54 @@
 import referee from "referee";
 import sinon from "../../lib/sinon";
 import * as aModule from "./a-module";
-const {assert} = referee;
-
-describe("Explicit lack of support for stubbing Modules", function() {
-    it("should give a proper error message for modules without default export", function() {
-        assert.exception(function() {
-            sinon.stub(aModule, "anExport").returns(400);
-        },
-        /TypeError: No support for stubbing ES Modules/);
+import aModuleWithDefaultExport from "./a-module-with-default";
+import functionModule, * as functionModuleAlternative from "./a-function-module";
+const {assert, refute} = referee;
+
+function createTestSuite(action){
+    var stub;
+    var errorRegEx = /TypeError: ES Modules cannot be (stubbed|spied)/;
+
+    afterEach(function() {
+        stub && stub.restore && stub.restore();
     });
-});
+
+    describe("sinon." + action + "()", function(){
+        describe("Modules with objects as their default export", function() {
+    afterEach(function() {
```

Comment:
Hopefully ESLint complains about indentation here ...

Reply:
No it did not, but I have now added a `lint-staged` script to catch this.